src/static/less/pages/homepage.less

```diff
@@ -658,6 +667,14 @@ section.video {
     }
   }
 
+  #about-the-rule {
+    h2 {
+      background-image: url( '../img/book-isocon.png');
```

Comment:
In Chrome and Firefox, I'm seeing the "getdata" icon here instead of the book icon.

<img width="1157" alt="screen shot 2015-10-05 at 10 43 29 am" src="https://cloud.githubusercontent.com/assets/1060248/10285226/d7590158-6b55-11e5-9bb4-22b2b8a8ead4.png">

After some investigating, I think this can be fixed by using the `.at2x` mixin here instead of directly specifying the background image:

```less
.at2x( '../img/book-isocon.png', 85px, 92px );
```